main/glossary/README.md

```diff
@@ -87,53 +95,45 @@ or `BigInt(123)`.
 The Board is a shared, on-chain location where users can post a value and make it
 accessible to others. When a user posts a value, they receive a unique ID for the value.
 Others can get the value just by knowing the ID. You can make an ID known by any
-communication method; private email, a DM or other private message, a phone call/voicemail,
-an email blast to a mailing list or many individuals, listing it on a website, etc.
+communication method&mdash;a DM or email or other private message, a phone call/voicemail,
+an email blast to a mailing list, publishing it on a website, etc.
```

Comment:
Note that these IDs are publicly visible, so publishing an object in the board makes it publicly accessible. If this is not appropriate for the object you have in mind, do not use the board to communicate access to that object.